spec/models/contact_info_spec.rb

```diff
@@ -40,28 +40,21 @@
   end
 
   context 'user emails' do
-    let!(:user1) { FactoryGirl.create(:user_with_emails, emails_count: 2) }
-    let!(:user2) { FactoryGirl.create(:user_with_emails, emails_count: 2) }
 
-    it "finds one user when there is one match" do
-      users = EmailAddress.where(:value => user1.contact_infos.first.value)
```

Comment:
did you take this out because it is weird to have a spec that tested a specific query?